### PR TITLE
Update Index.rst

### PR DESCRIPTION
**Pull request by averlon**
Set logging output

The directory is in 11.5 under var/log/

### CODE REVIEW

**Comment by linawolf** [comment]
Actually it depends on weather you have TYPO3 installed with or without composer. Should we list both possibilities?

**Comment by brotkrueml** [comment]
> Actually it depends on weather you have TYPO3 installed with or without composer. Should we list both possibilities?

I would opt for using tabs like for commands in this case.

File: Documentation/ApiOverview/Logging/Quickstart/Index.rst [hunk 1]
```diff
@@ -89,8 +89,8 @@ for warnings (:php:`LogLevel::WARNING`) and lower, so all matching log entries
 are written to a file.
 
 If the filename is not set, then the file will contain a hash like
-:file:`typo3temp/var/logs/typo3_<hash>.log`, for example
-:file:`typo3temp/var/logs/typo3_7ac500bce5.log`.
+:file:`var/log/typo3_<hash>.log`, for example
```
Comment (brotkrueml) [inline Documentation/ApiOverview/Logging/Quickstart/Index.rst:92]: ..  tabs::

    ..  group-tab:: Composer-based installation

        :file:`var/log/typo3_<hash>.log`, for example
        :file:`var/log/typo3_7ac500bce5.log`.

    ..  group-tab:: Legacy installation

        :file:`typo3temp/var/log/typo3_<hash>.log`, for example
        :file:`typo3temp/var/log/typo3_7ac500bce5.log`.

File: Documentation/ApiOverview/Logging/Quickstart/Index.rst [hunk 2]
```diff
@@ -89,8 +89,8 @@ for warnings (:php:`LogLevel::WARNING`) and lower, so all matching log entries
...
+:file:`var/log/typo3_7ac500bce5.log`.
```
Comment (brotkrueml) [inline Documentation/ApiOverview/Logging/Quickstart/Index.rst:93]:
```suggestion
```

File: Documentation/ApiOverview/Logging/Quickstart/Index.rst (hunk 1 above)
Comment (brotkrueml) [inline Documentation/ApiOverview/Logging/Quickstart/Index.rst:92]:
```suggestion
..  tabs::

    ..  group-tab:: Composer-based installation

        :file:`var/log/typo3_<hash>.log`, for example
        :file:`var/log/typo3_7ac500bce5.log`.

    ..  group-tab:: Legacy installation

        :file:`typo3temp/var/log/typo3_<hash>.log`, for example
        :file:`typo3temp/var/log/typo3_7ac500bce5.log`.

```

File: Documentation/ApiOverview/Logging/Quickstart/Index.rst [hunk 3]
```diff
@@ -89,8 +89,18 @@ for warnings (:php:`LogLevel::WARNING`) and lower, so all matching log entries
 are written to a file.
 
 If the filename is not set, then the file will contain a hash like
-:file:`typo3temp/var/logs/typo3_<hash>.log`, for example
-:file:`typo3temp/var/logs/typo3_7ac500bce5.log`.
+..  tabs::
```
Comment (brotkrueml) [inline Documentation/ApiOverview/Logging/Quickstart/Index.rst:92]:
```suggestion

..  tabs::
```

**Review by brotkrueml — APPROVED** [review]

**Review by linawolf — APPROVED** [review]

**Pull request merged**